### DISCUSSION TITLE
Share Your Website! 🌏

### DISCUSSION THREAD

**Discussion by javalosr2004** [hack4impact-calpoly/starter-pack#16] [Show and tell]
**This is not the Final Submission. 
If you are submitting for completion, go to the Final Submission discussion post!**

Post a link to your website below!

**Comment by KallosP** [comment]
My website's link is [here](https://KallosP.github.io); nothing too different yet. Even though I already have some experience with web development, I've learned a lot of useful good practices so far. I'm currently on lab 29.

**Comment by ljbarker** [comment]
Here's my website [link!](https://ljbarker.github.io) I have yet to add the styling, but the starter pack has been very informative so far!

**Comment by Shonueld** [comment]
[Here](https://shonueld.github.io/index.html) is my website so far! Haven't added any CSS yet, but it should be coming here soon, maybe by the time you click it!

**Comment by Firazaba0369** [comment]
Here is the [link](https://firazaba0369.github.io/index.html) to my website so far! I have not added styling but expect it very soon! I have learned a lot from the starter pack and am enjoying the experience.

**Comment by AlexBloomer** [comment]
Here is the [link](https://alexbloomer.github.io/resume.html) to my website so far! I haven't done much web development before so the starter pack was great!

**Comment by atkin0** [comment]
This is [my website](https://atkin0.github.io/index.html) so far, I forgot to upload to discussions so I have some CSS styling already.

**Comment by NolanKnievel** [comment]
[Here](https://nolanknievel.github.io/) is my website. I've never done any sort of web dev before but this is cool!

**Comment by vmagdangal** [comment]
As of posting, here's the link to [my site](https://vmagdangal.github.io) as of the end of lab 28. This has been a great refresher and I learned some new practices too! It should be updated sometime soon :)

**Comment by ChachaTeraSira** [comment]
Hello All,
This is the [link](https://chachaterasira.github.io/) to my website, it's not the best looking yet but I'm working on it. It is really awesome to visually see my code changing the website as I work on it. I'm not a web dev expert, this is my first time too, but I found something cool and efficient when working on website, if you work on VSCode, install live server extension, this will refresh the page on it's own and make chnages live when we make any changes to the code it self :)

**Comment by carboninl** [comment]
Hi! Here is my work-in-progress website: [link](https://carboninl.github.io/). Excited to start styling it!

**Comment by Taliavivrett** [comment]
Here is the [link](https://taliavivrett.github.io/) to my website! Definitely excited to make it pretty.

**Comment by amandayu255** [comment]
Here is the [link](https://amandayu255.github.io/) to my website.

**Comment by tranguyen-grc** [comment]
Here is the [link](https://tranguyen-grc.github.io/tranguyen.github.io/) to my website!

**Comment by samikshakarimbil** [comment]
[Here](https://samikshakarimbil.github.io/) is the link to my website

**Comment by arossi09** [comment]
You can check out my website so far [here](https://arossi09.github.io/)!

**Comment by dionnegregorio** [comment]
Here is my [website!](https://dionnegregorio.github.io/portfolio.html) have a look.

**Comment by blwelsh** [comment]
Here is my [personal website](https://blwelsh.github.io/index.html)! It's currently pretty bare, but I'm about to start the CSS section.

**Comment by otvalenc** [comment]
[HERE](https://leinotoo.github.io/index.html) is my website. Only just finished lab 27 and about to begin adding CSS.

**Comment by KevinBeltran23** [comment]
[HERE](https://kevinbeltran23.github.io/) is my personal website so far. I have yet to add any styling or actual content but I will get to that soon!

**Comment by 1kyler1** [comment]
[Here](https://1kyler1.github.io/portfolio.htmll) is a link to me website so far. Ive started to add a but of styling but im excited to give it the finishing touches.

**Comment by Amogh76** [comment]
[Here](https://amogh76.github.io/) is my website so far. There is still a lot of work to be done but I'm excited to continue.

**Comment by jasonyu0705** [comment]
[https://jasonyu0705.github.io](url) This is my website link.

**Comment by aleecodi** [comment]
https://crazzedman.github.io  Changes still being made to design, any feedback would be great.

**Comment by dmart297** [comment]
Here's [mine](https://dmart297.github.io/)! Have to add a picture for the portfolio part and design it all, but it's so fun so far!

**Comment by TylerHBE** [comment]
Here's the link to [mine](https://tylerhbe.github.io/index.html). Been fun so far!

**Comment by jaxonsprogis** [comment]
Here is the [link](jaxonsprogis.github.io) to my website. Feel free to take a look, at the moment it is pretty bare. I will be moving to the CSS section shortly.

**Comment by MekaSid** [comment]
Here is the [link](https://mekasid.github.io/) I'm excited to finish and style my website.

**Comment by Vishnu-Alachi123** [comment]
here is the [link](https://Vishnu-Alachi123.github.io) to my website, I haven't finished it yet, but it will be done soon.

**Comment by ian-ang-zhihan** [comment]
Here's the link to my [website ](https://ian-ang-zhihan.github.io/)- Ang-ineered by Ian!

**Comment by shaaronl** [comment]
here is a [link](https://shaaronl.github.io/index.html) to my website!  i still have to add styling, but the starter pack has been super informative and fun to follow along. excited for the final product :)

**Comment by koryyuen** [comment]
Here's a link to my [website](https://koryyuen.github.io/koryyuen/), I will probably still change the styling a bit.

**Comment by kevinrutledge** [comment]
Here's a [link](https://kevinrutledge.github.io/) to my website. I got a little carried away at this stage.

**Comment by darpank23** [comment]
Here's a [link](https://darpank23.github.io) to my website. I haven't done much yet in terms of styling, but follow along because I have some cool ideas for it!

**Comment by justinch4n** [comment]
Here's a [link to my website](https://justinch4n.github.io/index.html) so far! The CSS is a WIP, but I'm about to get into it.

**Comment by StearmanRubey** [comment]
Here is [my website!](https://stearmanrubey.github.io/l) I think everything has been going smoothly so far and I can't wait to stylize it.

**Comment by EthanSunjoonYang** [comment]
Here is my [website](https://ethansunjoonyang.github.io/)! Still got lots of work to do on it.

**Comment by IsaacPressman** [comment]
Here is my [website](https://isaacpressman.github.io/)! Haven't changed much so far, but still working on it

**Comment by hoikinlam** [comment]
Here is my [website](https://hoikinlam.github.io)! I'm currently working on it but it has been fun getting it to work.

**Comment by Edgarddragde** [comment]
Here is my [website](https://Edgarddragde.github.io)! I'm still working on it, it has been fun!

**Comment by Matthew-Le-027** [comment]
This is the [link](https://matthew-le-027.github.io/) to my website! There isn't anything that is too crazy, as I have only been following the instructions thus far, but website personalization will come soon! Also ignore the current picture on the home page, it is just a place holder for the time being.

**Comment by DashP06** [comment]
This is my [website](https://dashp06.github.io/Hack4Impact_Starter_Project/). Next is adding CSS to make it look nice.

**Comment by leili2000** [comment]
[Here is my website.](https://leili2000.github.io/) I'm mostly just laying out the framework for now and will be adding some more content later.

**Comment by karinaleee** [comment]
[Here](https://karinaleee.github.io/) is my link to my website! Go check it out

**Comment by Eclypsee** [comment]
[here](https://eclypsee.github.io/) is my site! it is live and polished! go check it out!

**Comment by Deep-Singh14** [comment]
[Here](https://deep-singh14.github.io/) is my website! I polished it slightly but plan on updating it in the next couple of days.

**Comment by BTK77** [comment]
[Here](https://btk77.github.io/) it is

**Comment by ivantorriani** [comment]
Here is my [website](ivantorriani.github.io). Still working on filling in all the pages...

**Comment by tomaseuu** [comment]
Here is my [website!](https://tomaseuu.github.io/thomas-l/) Hope you enjoy :)

**Comment by bbq27** [comment]
Here is my [website](https://bbq27.github.io/)! I'm still working on it but I've made some really good progress!

**Comment by zel701** [comment]
This is my [[[website]].](https://zel701.github.io/) I am still adding more information to it! But you can see me resume and my cat now!

**Comment by jonathanramirez4479** [comment]
Here is my [website](https://jonathanramirez4479.github.io/). Still working on the styling!

**Comment by ncoronac** [comment]
This is [my website](https://ncoronac.github.io/) so far! I've honestly loved working on this project and I'm excited to see how it will look in the end!

**Comment by allysonstauffer** [comment]
My website link is [here](https://allysonstauffer.github.io/). This project has been so fun to work on! I've also linked my repository [here](https://github.com/allysonstauffer/allysonstauffer.github.io).

**Comment by mcmike29** [comment]
My website is [here](https://mcmike29.github.io/). This has been really informative and I'm looking forward to using CSS next.

**Comment by lalog12** [comment]
[Website link](https://lalog12.github.io/index.html)
[Repository link](https://github.com/lalog12/lalog12.github.io)

Hope you like it!

**Comment by derrickph33** [comment]
This is my website progress so far: [Derrick's Website](https://derrickph33.github.io/)

**Comment by akshatk1001** [comment]
Here's the link to my website: [Akshat's Website](https://akshatk1001hack4impact.github.io)

**Comment by quinniedapooh** [comment]
[Here's my website so far.](https://quinniedapooh.github.io/) I'm really excited to finish with CSS!

**Comment by kailaaquino** [comment]
[Here's](https://kailaaquino.github.io/) how my website is looking so far. Theres no styling yet, pretty bare bones. Although I have done some web dev, it was fun to do all this from scratch and I learned a lot html tricks!

**Comment by ltrush** [comment]
As I'm writing this, it's lacking in style and behavior, but I'll see what I can get done before the deadline tonight. [Website Link](https://luketrusheim.github.io/resume.html)

**Comment by JavierMB03** [comment]
My website is [here](https://javiermb03.github.io/). I have never created a website before, but this has been a cool experience.

**Comment by Anivishy** [comment]
You can find my website [here](https://anivishy.github.io). Loved this experience and cant wait to incorporate css!

**Comment by TheGreatKiwi** [comment]
Here is the link to my website: file:///C:/Users/keira/thegreatkiwi.github.io/index.html
It is not very good, but this is my first time making one. I think I learned a lot from this!

**Comment by markmcguire011** [comment]
There's my [website](https://markmcguire011.github.io).

**Comment by jasmiineng** [comment]
Here's my [website](https://jasmiineng.github.io/) ! :)

**Comment by patstarzhou** [comment]
Hello!! Here is my website link! [website link](https://patstarzhou.github.io/patrick-z.github.io/index.html) Had a lot of fun making this!!

**Comment by wliu4040** [comment]
Here's my [website](https://wliu4040.github.io/index.html)

**Comment by qarteu** [comment]
hey heres my website https://qarteu.github.io/

**Comment by suesuee** [comment]
as of now: [website](https://suesuee.github.io/)

**Comment by lamkin1** [comment]
About to begin styling my website! https://lamkin1.github.io/

**Comment by anissasoungpanya** [comment]
https://anissasoungpanya.github.io/

**Comment by KailuanLiu** [comment]
https://kailuanliu.github.io/ looks a little messy but I can't wait to design it

**Comment by karthik0605** [comment]
https://karthik0605.github.io/

Still needs some design and CSS but I've learned a lot

**Comment by bri8679** [comment]
Here is my work in progress and ready to design https://bri8679.github.io/

**Comment by taikimorita** [comment]
Definitely still a work-in-progress but here's my website so far!
https://taikimorita.github.io/

**Comment by ewong128** [comment]
Still working on it, but I got some context up and have fun exploring [my website](https://ewong128.github.io/personalweb/index.html)

**Comment by lamkin1** [comment]
last minute submission: https://lamkin1.github.io/

**Reply by lamkin1** [reply]
Still working on a few things but mostly done! Here is the repository if interested to! https://github.com/lamkin1/lamkin1.github.io?tab=readme-ov-file

**Comment by mchn-4305** [comment]
You can visit my website here ! https://mchn-4305.github.io

**Reply by mchn-4305** [reply]
Sorry! Wrong link! Use this one! https://mchn-4305.github.io/
